packages/hoppscotch-common/src/components/http/Headers.vue

```diff
@@ -49,7 +49,7 @@
         />
       </div>
     </div>
-    <div v-if="bulkMode" class="h-full relative w-full">
+    <div v-if="bulkMode" class="h-full relative w-full flex flex-col flex-1">
```

Comment:
Can you confirm if you've cross-verified these additions with the changes in scope under #3848?

Reply:
We might also want to extend it to `HttpURLEncodedParams`.

https://github.com/hoppscotch/hoppscotch/blob/2917d50c6afe5f52fdcbb284514099d8423eca79/packages/hoppscotch-common/src/components/http/URLEncodedParams.vue#L47-L49

Reply:
> Can you confirm if you've cross-verified these additions with the changes in scope under https://github.com/hoppscotch/hoppscotch/pull/3848?

yup checked, working fine